keps/sig-api-machinery/2876-crd-validation-expression-language/README.md

```diff
@@ -589,6 +632,83 @@ associativeList.all(e, e.key1 == 'a' && e.key2 == 'b' && e.val == 100)
 associativeList.all(e, e.val == 100)
 ```
 
+### Access to Existing Object State
+
+#### Use Cases
+
+| Use Case                                                          | CEL Rule                                                                            | Examples |
+| --------                                                          | --------                                                                            | -------- |
+| Immutability                                                      | `self.foo == oldSelf.foo`                                                           ||
+| Prevent modification/removal once assigned                        | `oldSelf != 'bar' \|\| self == 'bar'` or `!has(oldSelf.field) \|\| has(self.field)` | [featureSet selection](https://github.com/openshift/kubernetes/blob/b426a3ffde5e7cd78483adeb5cb855f9f04dec5a/openshift-kube-apiserver/admission/customresourcevalidation/features/validate_features.go#L81-L88) |
```

Comment:
the `[featureSet selection]` link is surprising... can we drop this from the example since it's non-standard and somewhat orthogonal?